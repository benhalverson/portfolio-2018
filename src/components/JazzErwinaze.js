import React from 'react';
import jazz001 from '../images/Jazz-Erwinize001.png';
import jazz002 from '../images/Jazz-Erwinize002.png';
import jazz003 from '../images/Jazz-Erwinize003.png';
import jazz004 from '../images/Jazz-Erwinize004.png';
import jazz005 from '../images/Jazz-Erwinize005.png';
import jazz006 from '../images/Jazz-Erwinize006.png';
import jazz007 from '../images/Jazz-Erwinize007.png';
import jazz008 from '../images/Jazz-Erwinize008.png';
import jazz009 from '../images/Jazz-Erwinize009.png';

const JazzErwinaze = () => (
  <div>
 <div className="title">Jazz Pharmaceuticals</div>
          <div className="body">
            <p>
              {' '}
              This was a unique project becuae it took two different solutions
              that Viscira provided and combined them into one. I mostly worked
              on the Casebuilder part of this application.
            </p>
            <div className="screenshots">
              <img src={jazz001} alt="" width="100" height="100" />
              <img src={jazz002} alt="" width="100" height="100" />
              <img src={jazz003} alt="" width="100" height="100" />
              <img src={jazz004} alt="" width="100" height="100" />
              <img src={jazz005} alt="" width="100" height="100" />
              <img src={jazz006} alt="" width="100" height="100" />
              <img src={jazz007} alt="" width="100" height="100" />
              <img src={jazz008} alt="" width="100" height="100" />
              <img src={jazz009} alt="" width="100" height="100" />
            </div>
          </div>
  </div>
);

export default JazzErwinaze;