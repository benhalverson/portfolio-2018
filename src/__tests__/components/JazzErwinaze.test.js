import React from 'react';
import { shallow } from 'enzyme';
import JazzErwinaze from '../../components/JazzErwinaze';
test('it should render the JazzErwinaze component', () => {
  const wrapper = shallow(<JazzErwinaze />);
  expect(wrapper).toMatchSnapshot();
});
