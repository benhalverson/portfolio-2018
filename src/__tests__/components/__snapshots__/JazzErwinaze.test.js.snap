// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render the JazzErwinaze component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <JazzErwinaze />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="title"
>
          Jazz Pharmaceuticals
</div>,
        <div
          className="body"
>
          <p>
                     
                    This was a unique project becuae it took two different solutions that Viscira provided and combined them into one. I mostly worked on the Casebuilder part of this application.
          </p>
          <div
                    className="screenshots"
          >
                    <img
                              alt=""
                              height="100"
                              src="Jazz-Erwinize001.png"
                              width="100"
                    />
                    <img
                              alt=""
                              height="100"
                              src="Jazz-Erwinize002.png"
                              width="100"
                    />
                    <img
                              alt=""
                              height="100"
                              src="Jazz-Erwinize003.png"
                              width="100"
                    />
                    <img
                              alt=""
                              height="100"
                              src="Jazz-Erwinize004.png"
                              width="100"
                    />
                    <img
                              alt=""
                              height="100"
                              src="Jazz-Erwinize005.png"
                              width="100"
                    />
                    <img
                              alt=""
                              height="100"
                              src="Jazz-Erwinize006.png"
                              width="100"
                    />
                    <img
                              alt=""
                              height="100"
                              src="Jazz-Erwinize007.png"
                              width="100"
                    />
                    <img
                              alt=""
                              height="100"
                              src="Jazz-Erwinize008.png"
                              width="100"
                    />
                    <img
                              alt=""
                              height="100"
                              src="Jazz-Erwinize009.png"
                              width="100"
                    />
          </div>
</div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Jazz Pharmaceuticals",
          "className": "title",
        },
        "ref": null,
        "rendered": "Jazz Pharmaceuticals",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p>
               
              This was a unique project becuae it took two different solutions that Viscira provided and combined them into one. I mostly worked on the Casebuilder part of this application.
</p>,
            <div
              className="screenshots"
>
              <img
                            alt=""
                            height="100"
                            src="Jazz-Erwinize001.png"
                            width="100"
              />
              <img
                            alt=""
                            height="100"
                            src="Jazz-Erwinize002.png"
                            width="100"
              />
              <img
                            alt=""
                            height="100"
                            src="Jazz-Erwinize003.png"
                            width="100"
              />
              <img
                            alt=""
                            height="100"
                            src="Jazz-Erwinize004.png"
                            width="100"
              />
              <img
                            alt=""
                            height="100"
                            src="Jazz-Erwinize005.png"
                            width="100"
              />
              <img
                            alt=""
                            height="100"
                            src="Jazz-Erwinize006.png"
                            width="100"
              />
              <img
                            alt=""
                            height="100"
                            src="Jazz-Erwinize007.png"
                            width="100"
              />
              <img
                            alt=""
                            height="100"
                            src="Jazz-Erwinize008.png"
                            width="100"
              />
              <img
                            alt=""
                            height="100"
                            src="Jazz-Erwinize009.png"
                            width="100"
              />
</div>,
          ],
          "className": "body",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " ",
                "This was a unique project becuae it took two different solutions that Viscira provided and combined them into one. I mostly worked on the Casebuilder part of this application.",
              ],
            },
            "ref": null,
            "rendered": Array [
              " ",
              "This was a unique project becuae it took two different solutions that Viscira provided and combined them into one. I mostly worked on the Casebuilder part of this application.",
            ],
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt=""
                  height="100"
                  src="Jazz-Erwinize001.png"
                  width="100"
/>,
                <img
                  alt=""
                  height="100"
                  src="Jazz-Erwinize002.png"
                  width="100"
/>,
                <img
                  alt=""
                  height="100"
                  src="Jazz-Erwinize003.png"
                  width="100"
/>,
                <img
                  alt=""
                  height="100"
                  src="Jazz-Erwinize004.png"
                  width="100"
/>,
                <img
                  alt=""
                  height="100"
                  src="Jazz-Erwinize005.png"
                  width="100"
/>,
                <img
                  alt=""
                  height="100"
                  src="Jazz-Erwinize006.png"
                  width="100"
/>,
                <img
                  alt=""
                  height="100"
                  src="Jazz-Erwinize007.png"
                  width="100"
/>,
                <img
                  alt=""
                  height="100"
                  src="Jazz-Erwinize008.png"
                  width="100"
/>,
                <img
                  alt=""
                  height="100"
                  src="Jazz-Erwinize009.png"
                  width="100"
/>,
              ],
              "className": "screenshots",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "height": "100",
                  "src": "Jazz-Erwinize001.png",
                  "width": "100",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "height": "100",
                  "src": "Jazz-Erwinize002.png",
                  "width": "100",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "height": "100",
                  "src": "Jazz-Erwinize003.png",
                  "width": "100",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "height": "100",
                  "src": "Jazz-Erwinize004.png",
                  "width": "100",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "height": "100",
                  "src": "Jazz-Erwinize005.png",
                  "width": "100",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "height": "100",
                  "src": "Jazz-Erwinize006.png",
                  "width": "100",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "height": "100",
                  "src": "Jazz-Erwinize007.png",
                  "width": "100",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "height": "100",
                  "src": "Jazz-Erwinize008.png",
                  "width": "100",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "height": "100",
                  "src": "Jazz-Erwinize009.png",
                  "width": "100",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="title"
>
            Jazz Pharmaceuticals
</div>,
          <div
            className="body"
>
            <p>
                         
                        This was a unique project becuae it took two different solutions that Viscira provided and combined them into one. I mostly worked on the Casebuilder part of this application.
            </p>
            <div
                        className="screenshots"
            >
                        <img
                                    alt=""
                                    height="100"
                                    src="Jazz-Erwinize001.png"
                                    width="100"
                        />
                        <img
                                    alt=""
                                    height="100"
                                    src="Jazz-Erwinize002.png"
                                    width="100"
                        />
                        <img
                                    alt=""
                                    height="100"
                                    src="Jazz-Erwinize003.png"
                                    width="100"
                        />
                        <img
                                    alt=""
                                    height="100"
                                    src="Jazz-Erwinize004.png"
                                    width="100"
                        />
                        <img
                                    alt=""
                                    height="100"
                                    src="Jazz-Erwinize005.png"
                                    width="100"
                        />
                        <img
                                    alt=""
                                    height="100"
                                    src="Jazz-Erwinize006.png"
                                    width="100"
                        />
                        <img
                                    alt=""
                                    height="100"
                                    src="Jazz-Erwinize007.png"
                                    width="100"
                        />
                        <img
                                    alt=""
                                    height="100"
                                    src="Jazz-Erwinize008.png"
                                    width="100"
                        />
                        <img
                                    alt=""
                                    height="100"
                                    src="Jazz-Erwinize009.png"
                                    width="100"
                        />
            </div>
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Jazz Pharmaceuticals",
            "className": "title",
          },
          "ref": null,
          "rendered": "Jazz Pharmaceuticals",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                 
                This was a unique project becuae it took two different solutions that Viscira provided and combined them into one. I mostly worked on the Casebuilder part of this application.
</p>,
              <div
                className="screenshots"
>
                <img
                                alt=""
                                height="100"
                                src="Jazz-Erwinize001.png"
                                width="100"
                />
                <img
                                alt=""
                                height="100"
                                src="Jazz-Erwinize002.png"
                                width="100"
                />
                <img
                                alt=""
                                height="100"
                                src="Jazz-Erwinize003.png"
                                width="100"
                />
                <img
                                alt=""
                                height="100"
                                src="Jazz-Erwinize004.png"
                                width="100"
                />
                <img
                                alt=""
                                height="100"
                                src="Jazz-Erwinize005.png"
                                width="100"
                />
                <img
                                alt=""
                                height="100"
                                src="Jazz-Erwinize006.png"
                                width="100"
                />
                <img
                                alt=""
                                height="100"
                                src="Jazz-Erwinize007.png"
                                width="100"
                />
                <img
                                alt=""
                                height="100"
                                src="Jazz-Erwinize008.png"
                                width="100"
                />
                <img
                                alt=""
                                height="100"
                                src="Jazz-Erwinize009.png"
                                width="100"
                />
</div>,
            ],
            "className": "body",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " ",
                  "This was a unique project becuae it took two different solutions that Viscira provided and combined them into one. I mostly worked on the Casebuilder part of this application.",
                ],
              },
              "ref": null,
              "rendered": Array [
                " ",
                "This was a unique project becuae it took two different solutions that Viscira provided and combined them into one. I mostly worked on the Casebuilder part of this application.",
              ],
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt=""
                    height="100"
                    src="Jazz-Erwinize001.png"
                    width="100"
/>,
                  <img
                    alt=""
                    height="100"
                    src="Jazz-Erwinize002.png"
                    width="100"
/>,
                  <img
                    alt=""
                    height="100"
                    src="Jazz-Erwinize003.png"
                    width="100"
/>,
                  <img
                    alt=""
                    height="100"
                    src="Jazz-Erwinize004.png"
                    width="100"
/>,
                  <img
                    alt=""
                    height="100"
                    src="Jazz-Erwinize005.png"
                    width="100"
/>,
                  <img
                    alt=""
                    height="100"
                    src="Jazz-Erwinize006.png"
                    width="100"
/>,
                  <img
                    alt=""
                    height="100"
                    src="Jazz-Erwinize007.png"
                    width="100"
/>,
                  <img
                    alt=""
                    height="100"
                    src="Jazz-Erwinize008.png"
                    width="100"
/>,
                  <img
                    alt=""
                    height="100"
                    src="Jazz-Erwinize009.png"
                    width="100"
/>,
                ],
                "className": "screenshots",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "height": "100",
                    "src": "Jazz-Erwinize001.png",
                    "width": "100",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "height": "100",
                    "src": "Jazz-Erwinize002.png",
                    "width": "100",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "height": "100",
                    "src": "Jazz-Erwinize003.png",
                    "width": "100",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "height": "100",
                    "src": "Jazz-Erwinize004.png",
                    "width": "100",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "height": "100",
                    "src": "Jazz-Erwinize005.png",
                    "width": "100",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "height": "100",
                    "src": "Jazz-Erwinize006.png",
                    "width": "100",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "height": "100",
                    "src": "Jazz-Erwinize007.png",
                    "width": "100",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "height": "100",
                    "src": "Jazz-Erwinize008.png",
                    "width": "100",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "height": "100",
                    "src": "Jazz-Erwinize009.png",
                    "width": "100",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
